livekit-plugins/livekit-plugins-google/livekit/plugins/google/beta/realtime/realtime_api.py

```diff
@@ -382,7 +382,7 @@ def _on_input_speech_done(self, content: TranscriptionContent) -> None:
         # TODO: implement sync mechanism to make sure the transcribed user speech is inside the chat_ctx and always before the generated agent speech
 
     def _on_agent_speech_done(self, content: TranscriptionContent) -> None:
```

Comment:
when interrupted, are we only transcribing until the moment of interruption?

Reply:
got it. I think that's fine.. in the v1 branch, the synchronization/truncation logic will be downstream from the model.. model should just produce the entire thing.